### PR TITLE
#292 documentation updates

### LINKED ISSUE #292: Documentation updates
Add some links to the website; thinking to be linking to the github wiki. And repo, perhaps?
With the github repo, smarten that up; there's no description, website or licensing info provided.
Have updated the postman collection with URL (/v1) and the modified interactions query.

## Expected Behaviour
Should be able to find appropriate documentation when using the site.

## Current Behaviour
The documentation and postman collections are great, I think we just need to connect the items together.

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #292 

## Description
Adds some content (an [About](https://codes.msupply.foundation/about) page) with a link to it from the header and a link to the API docs in the footer.

Then we went a little offroad:
- Added a link to home on the icon (top left) because otherwise it is possible to get a little lost. It isn't obvious that the 'browse' link takes you to where you started, IMHO
- Added product details and link which is shown when viewing the Detail of a lower level entity. My concern was that after reaching the lower level detail page, it isn't clear how to go back to the page you were on

and I tweaked the github repo, to give an 'About' section and some tags. removed releases and packages, since they aren't very relevant. 

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [ ] Tests OK.

### CODE REVIEW

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+export type DetailProduct = React.FunctionComponent<DetailProductProps>;
+
+export const DetailProductComponent: DetailProduct = ({ entity, fetchEntity }) => {
+  const classes = useStyles();
+  const viewEntity = useViewEntity();
+
+  const { product } = entity ?? {};
+
```
Comment (wlthomson) [inline unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx:46]:
```suggestion
const { code, description } = product ?? {};
```

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  if (!product) return null;
```
Comment (wlthomson) [inline unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx:47]:
```suggestion
  if (!code) return null;
```

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  const onProductClick = () => {
+    if (!product.code) return;
+    fetchEntity && fetchEntity(product.code);
+    viewEntity(product.code);
+  };
```
Comment (wlthomson) [inline unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx:53]:
```suggestion
  const onProductClick = React.useCallback(() => {
    fetchEntity && fetchEntity(code);
    viewEntity(code);
  }, code);
```

Untested, but something like that?

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+    <Container className={classes.root}>
+      <Typography variant="subtitle1">Product</Typography>
+      <Typography className={classes.product} onClick={onProductClick}>
+        {product.description}
+        {product.code && <span>{product.code}</span>}
```
Comment (wlthomson) [inline unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx:60]:
```suggestion
        {description}
        {code && <span>{code}</span>}
```

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    if (!product.code) return;
+    fetchEntity && fetchEntity(product.code);
+    viewEntity(product.code);
+  };
+
+  return (
+    <Container className={classes.root}>
+      <Typography variant="subtitle1">Product</Typography>
+      <Typography className={classes.product} onClick={onProductClick}>
+        {product.description}
```
Comment (wlthomson) [inline unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx:59]: No `span` for `description`?

**Review by wlthomson — APPROVED** [review]
lgtm

File: unified-codes/apps/web/src/components/containers/about/AboutContent.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      <a href="https://xkcd.com/927/">https://xkcd.com/927/</a>
+    </Container>
+    <Container>
+      However, to our (limited) knowledge none of the existing universal codes do what we needed
+      them to do. Which is to:
+      <ul>
+        <li>Code items on their generic name, not their trade name (good-bye GTINs etc...)</li>
+        <li>
+          Code items based on their strength and dosage formulation (good-bye
+          https://www.unspsc.org/)
```
Comment (katherine-sussol) [inline unified-codes/apps/web/src/components/containers/about/AboutContent.tsx:20]: Could potentially link-ify this one too?

**Review by katherine-sussol — APPROVED** [review]
Looks good to me too, like the About page!

File: unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx (hunk 5 above)
Comment (mark-prins) [reply unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx]: I was sneakily trying to target the code without putting a class on it. I've done it better now.

**Pull request merged**